src/tbp/monty/frameworks/utils/logging_utils.py

```diff
@@ -809,6 +814,15 @@ def add_evidence_lm_episode_stats(lm, stats):
             ),
             4,
         )
+    # Check if the most likely object is a consistent child object
+    # Don't do this if the episode timed out, we had no match, or the detected object
+    # was already an exact match with the label.
+    if (
+        stats["primary_performance"] in ["confused", "confused_mlh"]
+        and consistent_child_objects is not None
```

Comment:
The argument `consistent_child_objects` seems to be a collection from it's use in this function. I think in that case it being None and it being empty should be handled the same and thus this can be simplified to:
```suggestion
        and consistent_child_objects
```